src/idl_gen_python.cpp

```diff
@@ -17,6 +17,7 @@
 // independent from idl_parser, since this code is not needed for most clients
 
 #include <cctype>
+#include <iostream>
```

Comment:
nix this

Reply:
oh yep, will do that left that in from my own debugging